src/components/Header/Header.js

```diff
@@ -90,6 +90,7 @@ class Header extends React.Component {
       patron,
       navData,
       currentTime = Date.now() || undefined,
+      isTest = false,
```

Comment:
Set the flag to determine if the codes are run for testing, so the tests won't have the unnecessary long timeout.

See `src/utils/encoreLogOutTimer.js` line 62, 63, 85